raiden-dapp/tests/unit/components/account/account-content.spec.ts

```diff
@@ -200,4 +200,36 @@ describe('AccountContent.vue', () => {
       }),
     );
   });
+
+  test('displays "Disconnect" button when connected', async () => {
+    const wrapper = await createWrapper(
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
+      undefined,
+    );
+    const disconnectButton = wrapper.find('.account-content__disconnect__button');
+
+    expect(disconnectButton.exists()).toBe(false);
+  });
+
+  test('does not display "Disconnect" button when not connected', async () => {
+    const wrapper = await createWrapper();
+    const disconnectButton = wrapper.find('.account-content__disconnect__button');
+
+    expect(disconnectButton.exists()).toBe(true);
+  });
+
+  test('Clicking "Disconnect" button triggers disconnect method', async () => {
+    const wrapper = await createWrapper();
+    (wrapper.vm as any).disconnect = jest.fn();
+    const disconnectButton = wrapper.find('.account-content__disconnect__button');
```

Comment:
Nitpick: you do this three times. In other unit tests we considered to export such as a function like `getDisconnectButton(wrapper)`. Just judge yourself if you think this is already worth here to add. I guess there are arguments for both here. :shrug:

Reply:
Done!